students/aminavi/session03/list_lab.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+list = ["Apple","Pears","Oranges","Peaches"]
```

Comment:
`list` is a built-in type name (for Python lists!), so it's not recommended to use it as your own variable name. Consider using something else, like `lst` or just `l`.